guidelines/sc/20/parsing.html

```diff
@@ -1,8 +1,31 @@
 <section class="sc">
    
-   <h4>Parsing</h4>
+   <h4>Parsing (Obsolete)</h4>
+
+   <p class="conformance-level">A</p>
+
+   <p>In content implemented using markup languages, elements have complete start and end
+      tags, elements are nested according to their specifications, elements do not contain
+      duplicate attributes, and any IDs are unique, except where the specifications allow
+      these features.
+   </p>
+
+   <div class="note">
+      <p>
+         Modern web technologies have standardized how user agents parse incorrect markup. 
+         Any invalid markup is therefore allowed under 4.1.1 Parsing for technologies such as
+         HTML 5 and CSS 3. This success criterion is always satisfied for these technologies. 
+         This success criterion is therefore obsolete.
+      </p><p>
+         Issues such as incorrect states or names due to a duplicate ID, or missing roles 
+         due to nesting interactive elements are covered by different success criteria.
+      </p>
+   </div>
+
+   <p class="note">Start and end tags that are missing a critical character in their formation, such
+      as a closing angle bracket or a mismatched attribute value quotation mark are not
+      complete.
+   </p>
 
    
-   <p>This criterion has been removed.</p>
-   
-</section>
\ No newline at end of file
+</section>
```

Comment:
Note I didn't update the understanding doc. If this goes through, the understanding doc needs to be tweaked. Happy to help with that if the group's open to this solution.